tests/test_analysis.py

```diff
@@ -1624,6 +1624,29 @@ def test_movement_onset_and_cursor_leave_time(self):
         self.assertTrue(np.all(cursor_leave_time == np.array([5,7])))
 
 class ControlTheoreticAnalysisTests(unittest.TestCase):
+    def test_get_machine_dynamics(self):
+        freqs = np.linspace(0,1,20)
+        exp_freqs = freqs[::2] # "even" freqs
+        print(freqs)
+        print(exp_freqs)
+        
+        M = controllers.get_machine_dynamics(freqs, 0)
```

Comment:
really minor -- could you add comments to what the test is? would help me figure out if the asserts make sense